pages/sentinel_create_samples.vue

```diff
@@ -99,15 +99,15 @@ export default {
       } else {
         this.items = []
       }
-      this.sampleCreationAlert(errored, successful)
+      this.sentinelSampleCreationAlert(errored, successful)
     },
-    sampleCreationAlert(errored, successful) {
+    sentinelSampleCreationAlert(errored, successful) {
       const msg = errored.map((e) => e.errors.join(', ')).join(', ')
-      if (errored.length && successful.length) {
+      if (errored.length > 0 && successful.length > 0) {
         this.showAlert(`Some samples were successfully created however: ${msg}`, 'warning')
-      } else if (errored.length) {
+      } else if (errored.length > 0) {
```

Comment:
`} else if (errored.length > 0 && successful.length === 0) {`

Reply:
it should never get to this if case if successful.length does not equal 0 as it is an else if block but i could add that to make it more explicit